File: CodeWars/Python/8Kyu/CatYearsDogYears.py
# Link: https://www.codewars.com/kata/cat-years-dog-years/train/python

def human_years_cat_years_dog_years(human_years):

    if human_years == 1:
        catYears = 15 
        dogYears = 15 
    elif human_years == 2:
        catYears = 15 + 9
        dogYears = 15 + 9
    else:
        catYears = 15 + 9 + human_years*4
        dogYears = 15 + 9 + human_years*5

    return [human_years,catYears,dogYears]

File: CodeWars/Java/8Kyu/CatYearsDogYears.java

public class CatYearsDogYears {
	public static int[] humanYearsCatYearsDogYears(final int humanYears) {
		int catYears;
		int dogYears;
		
		
		if (humanYears == 1){
	        catYears = 15; 
	        dogYears = 15;
		}
		else if (humanYears == 2){
	        catYears = 15 + 9;
	        dogYears = 15 + 9;
	    }
	    else {
	        catYears = 15 + 9 + (humanYears-2)*4;
	        dogYears = 15 + 9 + (humanYears-2)*5;
	    }
		
	    return new int[]{humanYears, catYears, dogYears};
	  }
	
	public static void main(String[] args) {
		int n = 1;
		System.out.println(humanYearsCatYearsDogYears(n));
	}
}
